src/components/TitleBar.js

```diff
@@ -4,23 +4,24 @@ import { Link } from 'src/components/common'
 import { icon } from 'src/components/icons'
 
 
-const TitleBar = ({ onPrevious, title, onDismiss, titleExtras }) => {
+const TitleBar = ({ onPrevious, title, onDismiss, titleExtras, titleSize = '0.875rem', arrowSize = '17', dismissSize = '20' }) => {
```

Comment:
Would appreciate input on this. Currently only the cluster config drawer and IGV drawer use this titlebar component. I added these extra props so could customize to match the mock for the cluster-config drawer and added defaults so that IGV drawer won't change. Wasn't sure if should have gone with a Wrapper approach instead? Or just had the IGV drawer also change?

Reply:
This feels like a lot of props for a relatively small component, which suggests that maybe the abstraction isn't quite right. The fundamental question is whether this is intended to be a consistent pattern, or just a specific layout for this case. (UX may have some opinions on that). If it's a pattern, then we should reduce the amount of customization and let the other use cases inherit the changes as well. If it's more one-off, then perhaps we should dissolve this component (perhaps in favor of a few smaller ones that could be composed together).

Reply:
Thanks for the guidance, I reached out to Lou and got confirmation that the style for the cluster drawer should be the new pattern to follow so I can simplify this/take out the customization